sources/packages/web/src/views/institution/AddInstitutionLocation.vue

```diff
@@ -57,7 +57,11 @@ export default {
           store.dispatch("institution/getUserInstitutionLocationDetails");
           snackBar.success("Institution Location created Successfully!");
         } catch (excp) {
-          snackBar.error("An error happened during the create process.");
+          if (excp instanceof ApiProcessError) {
```

Comment:
Please change the catch to `catch (error: unknown)`.

Reply:
ok. Done. Thanks!